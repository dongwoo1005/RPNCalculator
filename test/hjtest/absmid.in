0 ABS 0 NEG + 1222 - 0 *
