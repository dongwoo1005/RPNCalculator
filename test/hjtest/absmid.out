(((|0| + -0) - 1222) * 0)
= 0
